utils.py

```diff
@@ -220,6 +220,52 @@ def plot_mutations_per_node(self, max_num_muts=None, show_counts=False):
         )
         plt.xlabel("Number of mutations")
         plt.ylabel("Number of nodes")
+<<<<<<< HEAD
```

Comment:
Whoops, some conflicts to resolve here!

Reply:
Ah, thanks. I think I've resolved all conflicts now?!